src/libraries/System.Net.WebSockets.Client/src/System/Net/WebSockets/WebSocketHandle.Managed.cs

```diff
@@ -226,6 +246,72 @@ public async Task ConnectAsync(Uri uri, CancellationToken cancellationToken, Cli
             }
         }
 
+        private static WebSocketDeflateOptions ParseDeflateOptions(ReadOnlySpan<char> extension, WebSocketDeflateOptions original)
+        {
+            var options = new WebSocketDeflateOptions();
+
+            while (true)
+            {
+                int end = extension.IndexOf(';');
+                ReadOnlySpan<char> value = (end >= 0 ? extension[..end] : extension).Trim();
+
+                if (value.Length > 0)
+                {
+                    if (value.Equals(ClientWebSocketDeflateConstants.ClientNoContextTakeover, StringComparison.Ordinal))
```

Comment:
This can be just `value.SequenceEquals(ClientWebSocketDeflateConstants.ClientNoContextTakeover)`